cloudplatform/connectivity-oauth/src/main/java/com/sap/cloud/sdk/cloudplatform/connectivity/OAuth2Options.java

```diff
@@ -58,6 +69,15 @@ public final class OAuth2Options
     @Getter
     private final KeyStore clientKeyStore;
 
+    /**
+     * Configuration for caching OAuth2 tokens.
+     *
+     * @since 5.21.0
+     */
+    @Nonnull
+    @Getter( AccessLevel.PACKAGE )
```

Comment:
(Question)

Just wondering, why `package` scoped?

```suggestion
    @Getter
```

Reply:
Ah. I just thought this is something the user set themselves anyway and simply minimize public api. I will make it public.